dist/ramda.js

```diff
@@ -6200,10 +6198,6 @@
     };
 
     var R = {
-        F: F,
-        I: I,
-        T: T,
-        __: __,
```

Comment:
The build is failing because it *appears* you forgot to update __ramda.js__. You're right about the file system read. I'll open a pull request to add `R.sortBy` to the build script to make the output deterministic.